.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Enforce some consistency in files
+# https://editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+# Trim unnecessary whitespace from the end of lines
+# Use 2 spaces for indents (not tabs)
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
No idea what this file is for?

Reply:
I've removed for now. Think it could be a useful addition to the repo, but not really part of this PR.